### PR TITLE
feat(leavittbook): creates titanium-data-table-header demo

### LINKED ISSUE #252: Create a titanium-data-table-header demo
Read through props and make sure there are demos to cover everything.
Make sure the demos make sense and work well. Maybe style them a little bit.

### PR DESCRIPTION
**Pull request by carlycue**
Closes #252

### CODE REVIEW

**Review by SpencerMcFadden — APPROVED** [review]

File: packages/leavittbook/src/components/titanium-data-table-header.ts [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+@customElement('titanium-data-table-header-demo')
+export class TitaniumDateTableHeaderDemoElement extends LitElement {
+  static styles = [h1, h2, h3, h5, p, StoryStyles, css``];
+
+  #defaultStory() {
+    return html`
+      <div>
+        <h1>Default</h1>
+        <p>Examples using large, right, no-sort, and width attributes</p>
+        <titanium-data-table-header large column-name="Name" large title="Name"></titanium-data-table-header>
```
Comment (antwonthegreat) [inline packages/leavittbook/src/components/titanium-data-table-header.ts:20]: Can we have the Title reflect the property being demo'd? So this one would be "Large",
Also this has the large attribute twice

File: packages/leavittbook/src/components/titanium-data-table-header.ts [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+
+@customElement('titanium-data-table-header-demo')
+export class TitaniumDateTableHeaderDemoElement extends LitElement {
+  static styles = [h1, h2, h3, h5, p, StoryStyles, css``];
+
+  #defaultStory() {
+    return html`
+      <div>
+        <h1>Default</h1>
+        <p>Examples using large, right, no-sort, and width attributes</p>
```
Comment (antwonthegreat) [inline packages/leavittbook/src/components/titanium-data-table-header.ts:19]: ,desktop

File: packages/leavittbook/src/components/titanium-data-table-header.ts [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+        <titanium-data-table-header large column-name="Name" large title="Name"></titanium-data-table-header>
+        <titanium-data-table-header column-name="SName" desktop title="Short Name"></titanium-data-table-header>
+        <titanium-data-table-header column-name="person_type" right desktop title="Type"></titanium-data-table-header>
+        <titanium-data-table-header no-sort desktop center width="105px" title="Phone Number"></titanium-data-table-header>
```
Comment (antwonthegreat) [inline packages/leavittbook/src/components/titanium-data-table-header.ts:23]: Can we just do one property per header so it is clear what that property does?

Comment (carlycue) [reply packages/leavittbook/src/components/titanium-data-table-header.ts]: [68fe7c2](https://github.com/LeavittSoftware/titanium-elements/pull/336/commits/68fe7c20e33125a29cc362379f7baefa6ffd8a72)

**Review by antwonthegreat — APPROVED** [review]

**Review by aarondrabeck — APPROVED** [review]

File: packages/leavittbook/src/demos/titanium-data-table-header/titanium-data-table-header-demo.ts [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+        height: 900px;
+      }
+    `,
+  ];
+
+  render() {
+    return html`
+      <story-header
+        name="Titanium data table header"
+        tagName="titanium-data-table-header"
```
Comment (aarondrabeck) [inline packages/leavittbook/src/demos/titanium-data-table-header/titanium-data-table-header-demo.ts:25]:
```suggestion
    
```

File: packages/leavittbook/src/demos/titanium-data-table-header/titanium-data-table-header-demo.ts [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+        packageName="titanium-data-table"
+        klass="TitaniumDataTableHeaderElement"
```
Comment (aarondrabeck) [inline packages/leavittbook/src/demos/titanium-data-table-header/titanium-data-table-header-demo.ts:27]:
```suggestion
        className="TitaniumDataTableHeaderElement"
```

**Review by aarondrabeck — APPROVED** [review]

File: packages/leavittbook/src/demos/titanium-data-table-header/titanium-data-table-header-demo.ts (hunk 5 above)
Comment (carlycue) [reply packages/leavittbook/src/demos/titanium-data-table-header/titanium-data-table-header-demo.ts]: [77ed3ff](https://github.com/LeavittSoftware/titanium-elements/pull/336/commits/77ed3ff4a22b0748bce5048c9ce8069608f06a57)

File: packages/leavittbook/src/demos/titanium-data-table-header/titanium-data-table-header-demo.ts (hunk 4 above)
Comment (carlycue) [reply packages/leavittbook/src/demos/titanium-data-table-header/titanium-data-table-header-demo.ts]: [77ed3ff](https://github.com/LeavittSoftware/titanium-elements/pull/336/commits/77ed3ff4a22b0748bce5048c9ce8069608f06a57)

**Pull request merged**